### PR TITLE
[-] FO : Fix undefined label when using facet filters

### PR DESCRIPTION
**Pull request by maximebiloe**
| Questions | Answers |
| --- | --- |
| Branch? | develop |
| Description? | When you select a facet, you had a warning for undefined `label`. This PR fixes it and also put a border on selected color. |
| Type? | bug fix |
| Category? | FO |
| BC breaks? | no |
| Deprecations? | no |
| Fixed ticket? |  |
| How to test? | Go to a category page and select a color. You shouldn't have warnings anymore and have a border around the selected color. |

### CODE REVIEW

**Comment by vTerenti** [comment]
QA OK!

File: themes/classic/templates/catalog/_partials/facets.tpl [hunk 1]
```diff
@@ -9,7 +9,7 @@
             {foreach from=$facet.filters item="filter"}
               {if $filter.displayed}
                 <li>
-                  <label class="facet-label">
+                  <label class="facet-label{if $filter.active} active {/if}">
```
Comment (Shudrum) [inline themes/classic/templates/catalog/_partials/facets.tpl]: A space here would be more readable, too late I know.

```html
<label class="facet-label {if $filter.active}active{/if}">
```

**Comment by Shudrum** [comment]
Did this modification should and, if needed, is ported to the StarteTheme?

Thank you!

**Comment by maximebiloe** [comment]
The information to know if the filter is active is already present in the `.tpl`, I just moved it, just to respect the mockups. So, non need to do it in `StarterTheme` 😄

**Comment by Shudrum** [comment]
My bad… sorry & thank you!

**Pull request merged**